build/parent/pom.xml

```diff
@@ -34,7 +34,7 @@
         <upgrade-plan-content-zip-url>https://us-east-1.linodeobjects.com/devtools-s3.liferay.com/liferay-ide-files/docs/code-upgrade-docs-20210312.zip</upgrade-plan-content-zip-url>
         <upgrade-plan-content-zip-md5>c4bce64c362ecffcce300f31e88b2e63</upgrade-plan-content-zip-md5>
         <bndtools-site>https://bndtools.org/bndtools.p2.repo/7.0.0/</bndtools-site>
-        <eclipse-site>https://download.eclipse.org/releases/2023-12/202312061001/</eclipse-site>
+        <eclipse-site>https://download.eclipse.org/releases/2025-03/202503121000/</eclipse-site>
```

Comment:
if you bump it this much you will likely need to grab latest release of other stuff, like buildship and m2e

Reply:
ok I may back off that for now then